target/linux/ar71xx/config-4.4

```diff
@@ -113,6 +113,7 @@ CONFIG_ATH79_MACH_GL_INET=y
 CONFIG_ATH79_MACH_GL_MIFI=y
 CONFIG_ATH79_MACH_GS_MINIBOX_V1=y
 CONFIG_ATH79_MACH_GS_OOLITE=y
+CONFIG_ATH79_MACH_HIVEAP_121=y
```

Comment:
Take a look also here for example of how it should look like with current approach: https://git.lede-project.org/?p=lede/pepe2k/staging.git;a=commit;h=e74a4385271f684e892ec3c9311859e686956610